src/usdm-client.ts

```diff
@@ -196,48 +194,45 @@ export class USDMClient extends BaseRestClient {
     return this.get('fapi/v1/fundingInfo');
   }
 
-  /**
-   * @deprecated use get24hrChangeStatistics() instead (method without the typo)
-   */
-  get24hrChangeStatististics(
-    params?: Partial<BasicSymbolParam>,
-  ): Promise<ChangeStats24hr | ChangeStats24hr[]> {
-    return this.get24hrChangeStatistics(params);
-  }
```

Comment:
If you're going to delete this one (I'm ok with it, it's been deprecated a while), note it in the PR body as a breaking change and do a bigger version bump to v2.16